vector/src/main/java/im/vector/app/features/crypto/recover/BootstrapSharedViewModel.kt

```diff
@@ -255,9 +279,23 @@ class BootstrapSharedViewModel @AssistedInject constructor(
                     }
                 } else return@withState
             }
-//            is BootstrapActions.ReAuth                           -> {
-//                startInitializeFlow(action.pass)
-//            }
+            BootstrapActions.MigrationHandleKeyLost -> {
+                // delete the backups as key is lost
+                session.coroutineScope.launch {
+                    clearAllExistingBackups()
+                }
+                doesKeyBackupExist = false
```

Comment:
I wonder if this part should also be inside the coroutine so that we are sure to have a consistent state. Let's say there is an error when clearing the existing backups for instance, then we should not update the state and the `doesKeyBackupExist` field to `false` I guess?